grpc/client/src/main/java/io/helidon/grpc/client/GrpcChannelDescriptor.java

```diff
@@ -221,6 +229,7 @@ public Builder loadBalancerPolicy(String policy) {
          *
          * @see io.grpc.ManagedChannelBuilder#nameResolverFactory(io.grpc.NameResolver.Factory)
          */
+        // @ConfiguredOption
```

Comment:
The same as above.

Reply:
Yes, the code does not seem to use this hence why I commented it out, but forgot to remove it. Removing this.